### DISCUSSION TITLE
Commit from auto_fix does not trigger github actions

### DISCUSSION THREAD

**Discussion by Raigen** [wearerequired/lint-action#328] [Q&A] [answered]
Hi everyone, I am facing a problem when using the `auto_fix` feature. The feature itself works fine, the errors are fixed and there is a commit pushed to the pull request branch. The PR is also from a branch on the same repo, so no forked issue as well.
But the commit from the linter action does not trigger another round of testing. Our required testing step is not beeing triggered.
![image](https://user-images.githubusercontent.com/894799/144029144-c04690dc-3cee-4df7-a232-d4d669f02e1a.png)
I guess I have to adjust something on the github action trigger, but not sure. The commit seems to be just like all the other commits, except it is beeing done by a github action.
Maybe someone already solved this issue and can give me a hint what is wrong.

**Comment by ocean90** [comment, marked as answer]
Hey there, that's expected behaviour when using the default token.

> When you use the repository's `GITHUB_TOKEN` to perform tasks on behalf of the GitHub Actions app, events triggered by the `GITHUB_TOKEN` will **not** create a new workflow run.

Source: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow

You have to use [a PAT with appropriate permissions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and use it instead.

**Reply by Raigen** [reply]
Thanks for the answer!
